internal/pkg/mapper/relations/load_test.go

```diff
@@ -0,0 +1,30 @@
+package relations_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	. "github.com/keboola/keboola-as-code/internal/pkg/mapper/relations"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+)
+
+func TestRelationsMapperLoad(t *testing.T) {
```

Comment:
Su na to unit testy, .... vacsi zmysel to bude davat v dalsich PR, kde sa to realne pouzije.